src/main/java/sx/blah/discord/api/internal/DiscordVoiceWS.java

```diff
@@ -257,11 +257,13 @@ public void run() {
 	 * Sets up the thread for receiving audio packets from the Discord UDP Socket.
 	 */
 	private void setupReceiveThread() {
-		Runnable receiveThread = () -> {
+		(new Thread((Runnable) () -> {
```

Comment:
Thread needs to be assigned to a variable so that it can be interrupted / shutdown, no?